### DISCUSSION TITLE
Recommendation for readme.md

### DISCUSSION THREAD

**Discussion by bent-verbiage** [mangiucugna/json_repair#71] [Ideas]
First of all: great work, it's very useful!

Minor suggestion to add to the readme:
- `pip install json-repair`
- the types of errors that will be corrected. For example: I wasn't sure if text before the opening { or after the closing } - something that LLMs sometimes do - would be automatically removed.

**Comment by mangiucugna** [comment]
Thank you! I have updated README.md, it's not super comprehensive but it gives an idea of what the library is capable of.

**Reply by bent-verbiage** [reply]
Nice!
